src/app/components/header/header.component.ts

```diff
@@ -19,6 +19,9 @@ export class HeaderComponent implements OnInit {
 
   ngOnInit(): void {
     this.role$ = this.store.select(selectAccess);
+    if (localStorage.getItem('token')) {
+      this.store.dispatch(getUser());
+    }
   }
```

Comment:
please move this to app.component from header

Reply:
done